pnpm-lock.yaml

```diff
@@ -1,9 +1,4 @@
-lockfileVersion: 5.4
-
-patchedDependencies:
-  react-instantsearch-hooks-server@6.32.1:
-    hash: hiec5pcpgig2m3qcu64y3isqke
-    path: patches/react-instantsearch-hooks-server@6.32.1.patch
+lockfileVersion: 5.3
 
```

Comment:
Downgrading the lockfile version will break our Vercel deployment (see https://github.com/iFixit/react-commerce/pull/752). Make sure `pnpm --version` shows you version 7, and then run `pnpm install:all` and commit the result.

Reply:
If you're not on version 7, you could do `npm install -g pnpm@7` to install it. https://pnpm.io/installation#using-npm